ovirt_imageio/client/_ui.py

```diff
@@ -53,9 +59,21 @@ def draw(self, value, transferred, phase=None, last=False):
         return line.ljust(self._width, " ") + end
 
 
+class JsonFormat(OutputFormat):
+
+    def draw(self, value, transferred, phase=None, last=False):
+        return json.dumps({
+            'transferred': transferred,
+            'size': self.size,
```

Comment:
We don't know size at the start of the operation. Do you want to return None or invalid value until we know the size? Usually it is better not to return the values you don't know. The caller can easily handle non existing values.